index.html

```diff
@@ -4,21 +4,33 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="style.css">   
     <title>Hello, I am Adam Chang</title>
 </head>
 <body>
     <header class="head-container">
-        <img id="logo" src="Images/Logo.png" alt="logo" height="30"> 
-        <p>I am Adam</p>
-        <img src="Images/menu.svg" alt="">
-       
+         
+        <h2 id="name">Adam Chang</h2>
+        <img  id="menu-btn" src="Images/Union.png" alt="menu">   
 
         
     </header>
-<!-- <footer>
-    <a href='https://pngtree.com/so/red'>red png from pngtree.com/</a>
-</footer>     -->
+<section class="main-page">
```

Comment:
The section for the headline should have a background image. Please add the image as a background of the section.

Reply:
I have added the background image to the css, do I have to add it in the html instead?